packages/rocketchat-livechat/imports/server/rest/sms.js

```diff
@@ -44,6 +44,30 @@ RocketChat.API.v1.addRoute('livechat/sms-incoming/:service', {
 		sendMessage.message.msg = sms.body;
 		sendMessage.guest = visitor;
 
+		if (sms.hasMedia) {
```

Comment:
it will always enter on this `if` because `hasMedia` is always `true`, so it is not really needed.